src/ekka_autocluster.erl

```diff
@@ -87,7 +87,9 @@ maybe_run_again(App, JoinResult) ->
          , no_nodes_outside => NoNodesOutside
          }),
     case NodeInCluster andalso Registered andalso NoNodesOutside of
-        true  -> ok;
+        true -> 
+            ?LOG(notice, "discovery and join successfully", []),
```

Comment:
I have a question, in EMQX, the default log level is `warning`, so this log will not be seen, this can cause problems for users, as they can see the "discovery did not succeed; retrying", but not the cluster success log.
Any ideas ?

Reply:
we should use warning instead of notice?

Reply:
> we should use warning instead of notice?

I'm not sure, this is a successfully notice, but use `warning` level ?